### PR TITLE
Add recently added features list

### PR DESCRIPTION
**Pull request by kendrahavens**
This came from a need to easily search our growing feature list all in one place. Currently in the Visual Studio release notes one must navigate to several different pages to get an idea of what has been added recently.

@tmat @jinujoseph

### CODE REVIEW

**Comment by kendrahavens** [comment]
Yee! I don't have permissions to merge. Whoever can do the honors. :)

File: docs/README.md [hunk 1]
```diff
@@ -1,4 +1,125 @@
...
+ * "Using" statements can be added when code is pasted into a file. A code fix appears after pasting recognized code that prompts you to add relevant missing imports.
+ * You can now use Find All References (Shift-F12) and CodeLens to show results from Razor (.cshtml) files in .NET Core projects. You can then navigate to the identified code in the relevant Razor files.
+ * You will now receive a warning when running code analysis using FxCop. .NET Compiler analyzers are the recommended way to perform code analysis going forward. Read more on migrating to .NET compiler platform analyzers.
+
+## <a id="prod-15"></a> dev15
+
+### dev15 Summary
+* Visual Studio-based code cleanup via Format Document 
+* Refactorings: InvertIf, add parameter from method callsite, remove unnecessary parentheses, convert if-else assignment, completion for attributes and return to ternary conditional. 
+* Sourcelink support for GitLab, GitHub Enterprise and On-Premise TFS
```
Comment (tmat) [inline docs/README.md]: I don't think SourceLink should be listed here. It has nothing to do with Roslyn.

**Comment by tmat** [comment]
Actually, one more thing. Shouldn't this rather be under `ide` directory? It does not list compiler features (except for Portable PDBs, which should presumably be not listed).

File: docs/README.md (hunk 1 above)
Comment (kendrahavens) [reply docs/README.md]: Sure, I can take it out.

**Comment by kendrahavens** [comment]
I'm just gonna reiterate: I don't have permissions to merge this. Thanks for your help, all!

**Pull request merged**